### PR TITLE
grammar formatting

### PR DESCRIPTION
**Pull request by teverett**

### CODE REVIEW

**Comment by teverett** [comment]
@kdcro101 @KvanTTT 

ok with these formatting changes from Antlr4Formatter?

File: cql3/CqlLexer.g4 [hunk 1]
```diff
@@ -1,246 +1,1073 @@
+
 lexer grammar CqlLexer;
 
-LR_BRACKET: '(';
-RR_BRACKET: ')';
 
-LC_BRACKET: '{';
-RC_BRACKET: '}';
+LR_BRACKET
+   : '('
```
Comment (KvanTTT) [inline cql3/CqlLexer.g4]: I prefer one-line declaration for tokens (here and further):

```antlr
LR_BRACKET: '(';
```

It's shorter and more clear.

File: cql3/CqlLexer.g4 [hunk 2]
```diff
@@ -1,246 +1,1073 @@
...
+   ;
+
+
```
Comment (KvanTTT) [inline cql3/CqlLexer.g4]: Excess line break here and further (more than one line).

File: cql3/CqlParser.g4 [hunk 3]
```diff
@@ -23,150 +23,149 @@
...
 parser grammar CqlParser;
 
-options { tokenVocab=CqlLexer; }
+options
+   { tokenVocab = CqlLexer; }
 
 root
-    : cqls? MINUSMINUS? eof
-    ;
+   : cqls? MINUSMINUS? eof
```
Comment (KvanTTT) [inline cql3/CqlParser.g4]: Why only 3 spaces, not 4?

**Comment by teverett** [comment]
Fair enough. These are essentially issues with Antlr4Formatter.  They could
be reported as issues against that git project.

**Pull request merged**